### PR TITLE
add spicebush and chokeberries

### PR DESCRIPTION
**Pull request by NetSysFire**
#### Summary
Content "Added chokeberries and spicebush"

#### Purpose of change

More native flora! This is one small step towards making the forest into a forest as it is currently closer to an orchard.

Chokeberry is an incredibly tempting looking berry but is apparently very, very bitter. Which is why it got that name. Apparently birds and wildlife snack on it without issues, we do have such mutation trees but eliminating morale penalties for such foods is not possible yet as far as I know.

Spicebush is a bush with berries that can be used as spice. Apparently you can also brew tea with the leaves, but this I did not implement as we already have enough sources of tea.

#### Describe the solution

1. Add chokeberry shrub, item, and smoking result.
   - It is very bitter but it can be smoked. Native americans have used it in pemmican, supporting the statement that it is better smoked. [[1]](https://foragerchef.com/black-chokeberry-aronia-melanocarpa/) This also mentions its use as dye, so I added this there, too.
   - As you can see above, there are other ways of preparing chokeberry, too but I intentionally skipped those as its just fruit juice with more effort and sieving.
   - Nutrition info and the general web results suggest that chokeberries indeed have around 50 calories per 100g. [[2]](https://fddb.info/db/en/food/natural_product_aronia_berries_chokeberries/index.html).
2. Add spicebush shrub, item and smoking result.
   - This can partially replace black pepper and is ideally smoked to preserve it. [[3]](https://backyardforager.com/spicebush-berries-lindera-benzoin/).
   - Spicebush would also grow along rivers, but I am not going to blindly edit the hardcoded C++ river flora again. Not pushing my luck.
   - Yes, it should probably be powdered/crushed but I am skipping this step as it would pretty much only be tedium.
3. When working on chokeberries, I noticed cranberries were described as sour. They are most definitely not sour. They are bitter as fuck. I know because I was foolish enough to try a fresh one in real life. This is also why I adjusted the morale penalty. I'd only eat those raw if I was starving and even then I'd still reconsider.
4. General sources of missing plants: https://www.ecolandscaping.org/06/designing-ecological-landscapes/native-plants/more-edible-and-landscape-worthy-native-plants-of-new-england/ and https://www.ecolandscaping.org/09/developing-healthy-landscapes/ecological-landscaping-101/edible-and-landscape-worthy-native-plants-of-new-england/

#### Testing

- ✅ Game loads without any errors.
- ✅ Both shrubs spawn as expected and also get harvested correctly.
- ✅ Both items smoke correctly and can be used in the intended recipes.

#### Additional context

Two done. Dozens to go. Includes but is unfortunately (due to work needed) not limited to: at least three more fruit bearing trees: the nannyberry, the tupelo/black gum and black cherry (Prunus serotina). Maybe also chokecherry (not to be confused with choke*berries*). And some more bushes: foxgrape, add groundnut beans (addition to the existing plant), indian cucumber probably, maaaybe staghorn sumac and wild raisin/witherod and hobblebush plus wild strawberry. New England has a lot of virburnum/grape stuff. That and some attention to edible flowers (mostly results in more plant stalks I guess).

### CODE REVIEW

**Comment by TheShadowFerret** [comment]
> Apparently you can also brew tea with the leaves, but this I did not implement _as we already have enough sources of tea_.

This objection doesn't make sense, if it conforms to reality then it should exist 🤔

**Comment by NetSysFire** [comment]
There are so many plants that have leaves or twigs or stalks that are suitable for tea, sometimes with questionable medical effects, too. We already have a lot of tea and making yet another tea item feels like a waste of time. This needs a clean up PR so these all can be merged into a single herbal tea with no medical effects imo. I am busy travelling the next 4 days however and I consider it out of scope anyways.

File: data/json/items/comestibles/raw_fruit.json [hunk 1]
```diff
@@ -312,6 +312,18 @@
     "smoking_result": "dry_fruit",
     "vitamins": [ [ "vitC", "52200 μg" ], [ "iron", "2300 μg" ], [ "calcium", "55100 μg" ], [ "fruit_allergen", 1 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "chokeberries",
+    "name": { "str": "handful of chokeberries", "str_pl": "handfuls of chokeberries" },
+    "description": "A handful of black chokeberries.  They look similar to blackcurrant.  Big and juicy but actually exceedingly bitter.",
+    "copy-from": "cranberries",
+    "calories": "46",
```
[the remark this answers is not in the document]
Comment (NetSysFire) [reply data/json/items/comestibles/raw_fruit.json]: I wonder why this loaded just fine ingame.

[the remark this answers is not in the document]
Comment (Night-Pryanik) [reply data/json/items/comestibles/raw_fruit.json]: Maybe it was auto-converted to int?

[the remark this answers is not in the document]
Comment (NetSysFire) [reply data/json/items/comestibles/raw_fruit.json]: Probably. But it should not I think.

**Comment by Night-Pryanik** [comment]
> It is very bitter

I haven't tasted it myself, but sources I read says it has sweet and sour taste, not bitter one.

**Comment by NetSysFire** [comment]
"Astringent" may mean bitter, sometimes sour. The very reason chokeberry is named chokeberry is because it is mouth puckering in some way. Do note that some sources confuse other aronia species with black, wild chokeberry (Aronia melanocarpa) and any juice you find is probably sweetened to make it bearable. Plus also confusion of chokeberry vs chokecherry. I have not tasted it myself as I live in Europe, not the US, and have no access to those berries.

In the end this may just be a simple change in wording. But the cranberry, which is also described as astringent, is most definitely *very* bitter.

**Comment by logros13** [comment]
"eliminating morale penalties for such foods is not possible yet as far as I know."

the taste modifier cbm? (might have gotten the name wrong quoting from memory)

**Comment by NetSysFire** [comment]
It is about making people who crossed the bird threshold not having the morale penalties. Wildlife like birds enjoy the berries.

**Comment by Maleclypse** [comment]
I went ahead and started a PR to hopefully help with the existing edible flags

**Pull request merged**